src/mcts/search.cc

```diff
@@ -835,104 +866,96 @@ int SearchWorker::PrefetchIntoCache(Node* node, int budget) {
 
 // 4. Run NN computation.
 // ~~~~~~~~~~~~~~~~~~~~~~
-void SearchWorker::RunNNComputation() {
-  // This function is so small as to be silly, but its parent function is
-  // conceptually cleaner for it.
-  if (computation_->GetBatchSize() != 0) computation_->ComputeBlocking();
-}
+void SearchWorker::RunNNComputation() { computation_->ComputeBlocking(); }
```

Comment:
Is it, uh, worthwhile to remove this particular abstraction? It isn't really much of one lol. Totally up to you

Reply:
It all was done in a first place for support within-one-thread multiple games (for training games generation), and this function was the main reason to do that. That feature was never finished, but we'll need the same for tree detaching.

It will be:

```
  for all searches:
    Do steps 1..3
  for all searches:
    Do step 4.   (with network_st_batch, only last one will actually block)
  for all searches:
    Do steps 5..7
```

So step 4 is important enough to have a trivial (for now) abstration function.